### DISCUSSION TITLE
No module named carla

### DISCUSSION THREAD

**Discussion by Saddy21** [guardstrikelab/carla_apollo_bridge#40] [Q&A] [answered]
I am getting this error "**No module named carla**" when running:-
**_cd /apollo/cyber/carla_bridge
python carla_cyber_bridge/bridge.py_**
![Screenshot from 2023-01-12 14-59-27](https://user-images.githubusercontent.com/98169680/212030314-9b3a2d80-f39a-4f4d-9c4e-a69801d58847.png)

I checked the ip address of carla docker version it is using : **IPAddress: 172.18.0.2**.
![Screenshot from 2023-01-12 14-59-03](https://user-images.githubusercontent.com/98169680/212030441-d8fb5cd3-ff50-44f3-980a-6093413cfa6e.png)

While in the apollo cyber/setup.bash is using                     :**CYBER_IP=172.17.0.1**
![Screenshot from 2023-01-12 14-54-20](https://user-images.githubusercontent.com/98169680/212030493-156ce518-8a01-4369-a0f0-fca40e4b9db3.png)

Is it because of this this error or something else needs to be changed.I have followed all the steps in the documentation [.](https://github.com/guardstrikelab/carla_apollo_bridge)
Please help.

@guardstrikeo

**Comment by charles-lei** [comment, marked as answer]
I think it is a bug introduced by recently force push, this issue also occurs on my local box. Please try to override `src/carla_python/carla/dist` with your Carla's ‘PythonAPI/carla/dist’. @Saddy21
